WcaOnRails/app/controllers/competitions_controller.rb

```diff
@@ -151,6 +151,8 @@ def create
     if @competition.save
       flash[:success] = t('competitions.messages.create_success')
       redirect_to edit_competition_path(@competition)
+
+      @competition.update!(created_at: Time.now)
```

Comment:
I don't think you actually don't need this, Rails should manage the `updated_at` column for you. See http://api.rubyonrails.org/classes/ActiveRecord/Persistence.html#method-i-save.

Reply:
Yep, and it touches `created_at` on creation (http://guides.rubyonrails.org/active_record_basics.html#schema-conventions)